app-2/src/main/java/tech/relaycorp/letro/awala/message/Message.kt

```diff
@@ -0,0 +1,6 @@
+package tech.relaycorp.letro.awala.message
+
+data class Message(
```

Comment:
I think we shouldn't replace Awala's `IncomingMessage` or `OutgoingMessage` with this because we will need additional fields from those classes pretty soon, even if we don't need them now.

For example, we're going to have to read `IncomingMessage.sender` to make sure it matches the third-party endpoint we expect -- because otherwise a rogue contact could impersonate the Letro server.

Reply:
It's an abstraction that is used in our app with fixed MessageType. In other words, we are not depending on the implementation of classes of Awala SDK and use our own abstraction

It won't be a problem to add a field to this class when we'll need that

Reply:
OK, fair enough 👍🏾